.eslintrc.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
       },
     ],
     "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/triple-slash-reference": "off",
```

Comment:
This rule was throwing an error for `/// <reference path="../support/index.d.ts" />` tags we place in the cypress tests this suppresses that error.